es/.gitignore

```diff
@@ -0,0 +1,20 @@
+.idea/
+*.iml
+*.iws
+*.eml
+out/
+.DS_Store
+.svn
+log/*.log
+tmp/**
+node_modules/
+.sass-cache
+.github
+css
+dist
+examples
+js
+plugin
+gulpfile.js
+package.json
+package-lock.json
```

Comment:
I'm also fine with all these being added to main .gitignore

Reply:
will move it on next PR